src/Content/Text/BBCode.php

```diff
@@ -995,29 +995,46 @@ private static function interpolateSavedImagesIntoItemBody($uriid, $body, array
 	}
 
 	/**
-	 *
-	 * @param  string   $text     A BBCode string
-	 * @return array share attributes
+	 * @param string $text A BBCode string
+	 * @return array Empty array if no share tag is present or the following array, missing attributes end up empty strings:
+	 *               - comment: Text before the opening share tag
+	 *               - shared : Text inside the share tags
+	 *               - author : (Optional) Display name of the shared author
+	 *               - profile: (Optional) Profile page URL of the shared author
+	 *               - avatar : (Optional) Profile picture URL of the shared author
+	 *               - link   : (Optional) Canonical URL of the shared post
+	 *               - posted : (Optional) Date the shared post was initially posted ("Y-m-d H:i:s" in GMT)
+	 *               - guid   : (Optional) Shared post GUID if any
 	 */
-	public static function fetchShareAttributes($text)
+	public static function fetchShareAttributes(string $text): array
```

Comment:
This seems like a perfect Situation for unit tests, doesn't it? ;-)

Reply:
Yes!

Reply:
And I just missed this one. Could catch!

Reply:
If you can, please put a method description up here.